hazelcast/src/main/java/com/hazelcast/instance/Node.java

```diff
@@ -210,6 +214,24 @@ public Node(HazelcastInstanceImpl hazelcastInstance, Config config, NodeContext
         }
     }
 
+    private ClassLoader getConfigClassloader(Config config) {
+        DynamicClassloadingConfig dynamicClassloadingConfig = config.getDynamicClassloadingConfig();
+        ClassLoader classLoader;
+        if (dynamicClassloadingConfig.isEnabled()) {
+            ClassLoader parent = config.getClassLoader();
+            final ClassLoader theParent = parent == null ? Node.class.getClassLoader() : parent;
+            classLoader = doPrivileged(new PrivilegedAction<DynamicClassLoader>() {
```

Comment:
well, findbugs was complaining. I could either add a suppression or do this. 
what do you see as a better option?